msu/hooks/entity/tactical/player.nut

```diff
@@ -27,6 +27,16 @@
 		this.m.LevelUpsSpent++;
 	}
 
+	q.onHired = @(__original) function()
+	{
+		__original();
+
+		foreach (bro in ::World.getPlayerRoster().getAll())
+		{
+			bro.getSkills().onAnyoneHired(this);
```

Comment:
onAnyoneHired here, onAnyBroHired later.
Personally I also think that onBroHired is good enough.

Reply:
I removed the left-over `onAnyBroHired` here.
If you prefer another term over `onAnyBroHired` you gotta convince midas